website/content/en/status/report-2023-04-2023-06/capsicum-ktracing.adoc

```diff
@@ -0,0 +1,251 @@
+=== Security Sandboxing Using `ktrace(1)`
+
+Links: +
+link:https://github.com/jakesfreeland/freebsd-src/tree/ff/ktrace[ktrace branch] URL: https://github.com/jakesfreeland/freebsd-src/tree/ff/ktrace +
```

Comment:
For consistency with other reports, the URL: bit should be a link too, like so: `URL: link:https://github.com/jakesfreeland/freebsd-src/tree/ff/ktrace[]`